features/admin_associates_customer_with_auction.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Admin associates customer with auction
```

Comment:
if below scenario is moved to create / update feature spec file, perhaps the other specs can be moved to files related to seeing auction attributes? I think that is how we are mostly dealing with our other attrs....